### PR TITLE
fixes quote import for py2

### PR DESCRIPTION
**Pull request by brwnj**

### CODE REVIEW

File: src/samplot_vcf.py [hunk 1]
```diff
@@ -15,10 +15,7 @@
 import os
 import random
 from collections import defaultdict
-try:
-    from shlex import quote
-except ImportError:
-    from pipes import quote
+from pipes import quote
```
Comment (brwnj) [inline src/samplot_vcf.py]: Takes advantage of https://github.com/python/cpython/blob/3.7/Lib/pipes.py#L65

**Comment by ryanlayer** [comment]
Can you update the requirements.txt file to include these packages so that the tests don't fail?

**Comment by brwnj** [comment]
Will do.

**Comment by jbelyeu** [comment]
merging this into a new test suite

**Pull request closed without merging**